x/protorev/keeper/posthandler.go

```diff
@@ -47,12 +48,12 @@ func (protoRevDec ProtoRevDecorator) PostHandle(ctx sdk.Context, tx sdk.Tx, simu
 	cacheCtx = cacheCtx.WithGasMeter(upperGasLimitMeter)
 
 	// Check if the protorev posthandler can be executed
-	if err := protoRevDec.ProtoRevKeeper.AnteHandleCheck(cacheCtx); err != nil {
+	if err := protoRevDec.ProtoRevKeeper.AnteHandleCheck(ctx); err != nil {
 		return next(ctx, tx, success, simulate)
 	}
 
 	// Extract all of the pools that were swapped in the tx
-	swappedPools := protoRevDec.ProtoRevKeeper.ExtractSwappedPools(cacheCtx)
+	swappedPools := protoRevDec.ProtoRevKeeper.ExtractSwappedPools(ctx)
```

Comment:
Why did these two change? Does it not work if this isn't being done?

Reply:
This was a mistake I made while commiting, chagned on the latest commit